File: 프로그래머스/0/120912. 7의 개수/7의 개수.cs
using System;

public class Solution 
{
    public int solution(int[] array) 
    {
        int count = 0;
        for(int i=0; i<array.Length; i++) {
            string str = array[i].ToString();
            for(int j=0; j<str.Length; j++) {
                if(str[j].ToString() == "7") {
                    count++;
                }
            }
        }
        return count;
    }
}

File: 프로그래머스/0/120912. 7의 개수/7의 개수.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(vector<int> array) 
{
    string temp = "";
    for(int &i : array)
    {
        temp += to_string(i);
    }
    
    int count = 0;
    for(char &c : temp)
    {
        if(c == '7')
        {
            count++;
        }
    }
    return count;
}